packages/java/tests/gradle/single-module-tests/package.json

```diff
@@ -18,17 +18,18 @@
     "@vaadin/router": "1.7.5",
     "classnames": "^2.3.2",
     "construct-style-sheets-polyfill": "3.1.0",
-    "lit": "3.1.2",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-router-dom": "6.22.2"
+    "lit": "3.2.1",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "react-router-dom": "6.22.2",
```

Comment:
Just a note: this version of the `react-router-dom` is outdated (same in the other Gradle IT project), with `"6.28.2"` being the latest. I have no good ideas why isn't it broken in the builds considering the `react` and `react-dom` are upgraded to the `18.3.1`. 
This version conflict was the reason of snapshot builds failure for 24.5 and 24.6. The curious part is that it was only happening in the `react-signals` ITs, and not for example in `react-grid-test` ITs (with an outdated version of `react-router-dom`).